ignite/templates/app/files-consumer/app/ante_handler.go.plush

```diff
@@ -0,0 +1,71 @@
+package app
```

Comment:
The wasm app also adds an `ante_handler.go` file into the app folder. Maybe we should find a way not to overwrite or throw an error. But I think we can open a new issue for it

Reply:
Sounds reasonable to create a new issue for it.